### PR TITLE
fix: TransferTransaction.Freeze() nil pointer panic, implemented vali…

### PR DESCRIPTION
**Pull request by andrix10**
…dateNetworkOnIDs nil client check

Signed-off-by: Andrei Kuzmiankov <andrei@launchbadge.com>

Closes: #246

### CODE REVIEW

File: account_create_transaction.go [hunk 1]
```diff
@@ -142,6 +142,9 @@ func (transaction *AccountCreateTransaction) GetAccountMemo() string {
 }
 
 func (transaction *AccountCreateTransaction) validateNetworkOnIDs(client *Client) error {
+	//if client == nil {
```
Comment (danielakhterov) [inline account_create_transaction.go:145]: I think you forgot to un-comment this.

**Review by danielakhterov — CHANGES_REQUESTED** [review]

**Comment by xin-hedera** [comment]
Can we add unit tests?

File: topic_message_submit_transaction.go [hunk 2]
```diff
@@ -127,6 +127,9 @@ func (transaction *TopicMessageSubmitTransaction) SignWith(
 }
 
 func (transaction *TopicMessageSubmitTransaction) validateNetworkOnIDs(client *Client) error {
+	if client == nil {
+		return nil
+	}
 	if !client.autoValidateChecksums {
```
Comment (steven-sheehy) [inline topic_message_submit_transaction.go:133]: Can't these be combined for succinctness?
```suggestion
	if client == null || !client.autoValidateChecksums {
```

Comment (mehcode) [reply topic_message_submit_transaction.go]: First mistake is trying to be succinct in Go. :laughing:

**Review by danielakhterov — APPROVED** [review]

**Comment by danielakhterov** [comment]
Created issue for unit tests: #255

**Pull request merged**